src/transactional_object_store.rs

```diff
@@ -74,7 +74,9 @@ impl TransactionalObjectStore for DelegatingTransactionalObjectStore {
 
     async fn get(&self, path: &Path) -> Result<GetResult, Error> {
         let path = self.path(path);
-        self.object_store.get(&path).await
+        self.object_store
+            .get_opts(&path, CacheableGetOptions::default())
```

Comment:
Same question here. I don't think we want to use default opts for get calls... We want to allow rank and cache settings don't we? We're only using the transactional store in manifest right now, but the plan is to use CAS for the WAL and compacted SSTs as well.